### ISSUE TITLE
Heads up: FreeBSD port is on its way

### ISSUE THREAD

**Issue by 0mp** [kusalananda/shell-toolbox#3]
Hello,

I just wanted to say that [a FreeBSD port of shell-toolbox has been submitted and is awaiting review and approval](https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=227537).

I've tested `shell` on FreeBSD a little bit and I confirm it works.

Cheers!

**Comment by kusalananda** [comment]
Oh, that's... unexpected (the port that is, not that it works on FreeBSD)  :-)

I've made some adjustments in the last couple of days on the `develop` branch, but it doesn't affect the workings of the `shell` script much (added a `-v` option for version info and made the `configure` script figure out whether we need to add `.XXXXXXXX` to the filename template when using `mktemp -t` (how does this work on FreeBSD? Do you get a working directory with `XXXXXXXX` in it when using the `master` code?)).

I was planning to make a new release in the weekend, but I could easily make one during the week to make sure FreeBSD picks up a version I'm happy with (with the changes from the last couple of days included). But you could go with the version you've picked if it works, no worries.

I see you have `USES=libtool` in the port. I'm not compiling anything, so this could _possibly_ go.

**Comment by 0mp** [comment]
On the latest release I get:

```
$ shell
Starting /usr/local/bin/bash in /tmp/shell-bash.XXXXXXXX.sLe2NoCy
```

I get the same behavior on master.

Yeah, `libtool` is not required there. Thanks ;)

No need to make the release earlier. I'll just update the port when the new release comes out 👍

**Comment by kusalananda** [comment]
Yeah, that directory name is not ideal. I'll set up a FreeBSD virtual machine to make user it works before making the next release.

Ok, I'll assume that you're watching.

Thanks for being interested!  👍
